### PR TITLE
Remove inline configs, create pw executor helper

### LINKED ISSUE #6: Read configuration from file plus additional fixes
@intsco please resubmit the pull request against this commit https://github.com/metaspace2020/pywren-annotation-pipeline/commit/68d4d37a967b61cb12600582cbb37857ab04ba1e

### PR DESCRIPTION
**Pull request by intsco**
Resolves #6

### CODE REVIEW

**Comment by intsco** [comment]
@gilv merging Jupyter notebooks is a bit trickier than source code. Hence more chances to miss something. Here is my second try

File: pywren-annotation-example-pipeline.ipynb [hunk 1]
```diff
@@ -97,57 +97,43 @@
...
-    "    'ibm_cf':  {'endpoint': 'https://eu-gb.functions.cloud.ibm.com',\n",
-    "              'namespace': 'kovalev@embl.de_dev', \n",
-    "              'action_memory' : 1024,\n",
-    "              'api_key': '***'}, \n",
-    "    'ibm_cos': {\n",
-    "        'endpoint': 'https://s3.eu-gb.cloud-object-storage.appdomain.cloud',\n",
-    "        'api_key' : '***'\n",
-    "    },\n",
-    "    'pywren' : {'storage_bucket' : 'metaspace-annotation-example',\n",
-    "                'runtime' : 'ibmfunctions/pywren-metabolomics:3.6'}\n",
```
Comment (gilv) [inline pywren-annotation-example-pipeline.ipynb:149]: @intsco did you updated your configuration file with 'runtime' : 'ibmfunctions/pywren-metabolomics:3.6' ?

**Comment by gilv** [comment]
@intsco when the config is in the external file, how do you plan to share this notebook with others? the notebook is not "stand alone" anymore, but requires external config file

File: pywren-annotation-example-pipeline.ipynb (hunk 1 above)
Comment (intsco) [reply pywren-annotation-example-pipeline.ipynb:149]: I create it using the same name convention a few lines below in `create_pw`

**Comment by intsco** [comment]
@gilv I don't think sharing the config with my credentials with anyone else is good idea anyway.
Are we aiming at this notebook to be standalone? My initial idea was to provide you with a prototype that you can modify slightly and share with us. Correct me if I'm wrong

**Comment by gilv** [comment]
@intsco if i understand correctly then @theodev wanted to share this notebook within your organization to get other inputs...if this is the case the notebook should be stand alone, with "'api_key': '***" and everybody can fill the right key. If this notebook is used by you only, then you can use external configuration files. I don't care...it's your call what you plan to do with this notebook ;) I just raised a concern if it should be stand alone or not

**Comment by intsco** [comment]
@gilv thanks for a heads up, we'll keep this in mind

**Pull request closed without merging**